torch/csrc/jit/codegen/cuda/codegen.cpp

```diff
@@ -446,6 +446,11 @@ class CudaKernelGenerator : private OptOutConstDispatch {
     code_ << varName(ti->view()) << "[" << index.str() << "]";
   }
 
+  void handle(const SplitVector* sv) final {
+    indent() << gen(sv->output(0)) << " = " << gen(sv->input(0)) << "["
+             << gen(sv->vector_id()) << "];\n";
```

Comment:
`gen(sv->vector_id())` was a bit mystery to me at first as I thought `vector_id()` always returns `IterDomain*`.

Reply:
This is now changed into `gen(sv->index())`.